nix/lib.nix

```diff
@@ -23,7 +23,7 @@ rec {
     buildInputs = [ git ];
     installPhase = ''
       mkdir -p $out
-      git clone --branch hackage https://github.com/commercialhaskell/all-cabal-hashes.git $out
+      git clone --bare --branch hackage https://github.com/commercialhaskell/all-cabal-hashes.git $out
```

Comment:
@binarin does `--bare` imply that the result of git clone is reproducible, and we can actually use `fetchgit` instead?

Reply:
Ok, I see. Nice improvement nevertheless :+1: